src/components/App.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display(props) {
+  const { numberDisplay } = props;
+  return (<div>{ numberDisplay }</div>);
+}
+
+Display.propTypes = {
+  numberDisplay: PropTypes.string,
+};
+
+Display.defaultProps = {
+  numberDisplay: '0',
+};
```

Comment:
It is placed in `src/components` directory ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/addToComponent.png)). It asks you to create a separate file for each component.